commit 571f09f8c72823a26f317800c24d50474049cd0c (refs/changes/82/5682/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-03-01T08:28:21-08:00 (7 months ago)
    
    TRITON-1267 workflow crashing due to a bad wf_job_overhead_seconds calculation

diff --git a/package.json b/package.json
index 7606314..908bd78 100644
--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "uuid": "3.0.1",
     "vasync": "1.6.4",
     "verror": "1.6.0",
-    "wf": "git+https://github.com/joyent/node-workflow.git#fcbd66058d290fa8284a25bda854ff8743478b4f",
+    "wf": "1.2.2",
     "wf-moray-backend": "1.2.0"
   },
   "devDependencies": {
